common/shared.c

```diff
@@ -363,7 +363,7 @@ int ft_init_av(void)
 		ret = fi_av_insert(av, fi->dest_addr, 1, &remote_fi_addr, 0, NULL);
 		if (ret != 1) {
 			FT_PRINTERR("fi_av_insert", ret);
-			return ret;
+			return -1;
```

Comment:
I had a second thought about the check it's not necessarily 0 all the time. Anytime av_insert returns < number of addresses to insert and in some tests we have more that one address to insert. But I can create a generic error message and return -1.

Agreed on the second part. Will update the PR.